src/integrationTest/java/uk/gov/hmcts/divorce/caseworker/CaseworkerIssueApplicationTest.java

```diff
@@ -157,7 +157,7 @@ void shouldGenerateAosPackAndPostAosPackAndSendEmailsWhenRespondentIsRepresented
         );
 
         when(serviceTokenGenerator.generate()).thenReturn(TEST_SERVICE_AUTH_TOKEN);
-        when(documentIdProvider.documentId()).thenReturn("Divorce application").thenReturn("Respondent Invitation");
+        when(documentIdProvider.documentId()).thenReturn("Respondent Invitation").thenReturn("Divorce application");
```

Comment:
Invitation to invitation ?

Reply:
These were the wrong way round, the Respondent Invitation is generated first